packages/bento-design-system/src/Charts/Config.ts

```diff
@@ -0,0 +1,7 @@
+import { allColors } from "../util/atoms";
+
+export type ChartDataColor = keyof typeof allColors;
```

Comment:
shouldn't we use only the dataViz colors here?

Reply:
I discussed this with Agnese, and she mentioned these can be any color. We'll probably iterate a bit more on this, allowing also special palettes for charts, but this is a good starting point I think.